main.js

```diff
@@ -4,13 +4,14 @@ const BrowserWindow = electron.BrowserWindow
 const path = require('path')
 const url = require('url')
 const fs = require('fs')
-const autoUpdater = require('electron-updater').autoUpdater
-const autoUpdaterInterval = 60 * 60 * 1000;
 const ipc = electron.ipcMain;
 const Menu = electron.Menu;
 const shell = electron.shell;
 const ElectronConfig = require('electron-config');
 
+const { initializeAutoUpdater } = require('./autoupdate');
+const { getLocaleMessages, normalizeLocaleName } = require('./locale');
```

Comment:
Also if code gets pulled out, it might be nice to put it in a sub-folder within the project root as well, if y'all have thoughts on that